### ISSUE TITLE
Constant errors - generic error: can only concatenate list (not "tuple") to list

### ISSUE THREAD

**Issue by JOHLC** [ljmerza/ha-email-sensor#56]
## Version of the custom_component
3.11.0

## Configuration

```yaml
sensor: 
  - platform: email
    email: !secret mailman_user
    password: !secret mailman_pass
    imap_server: !secret mailman_server
    days_old: "3"
```

## Describe the bug
An error is logged every few seconds per below. Sensor still appears to be functional. 

## Debug log

```text

2023-06-20 19:41:00.547 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.640 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.657 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.753 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.837 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.917 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.981 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:00.997 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.016 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.037 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.052 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.064 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.115 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.132 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.146 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.159 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.176 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.227 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.242 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.259 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list
2023-06-20 19:41:01.272 ERROR (SyncWorker_6) [custom_components.email.sensor] generic error: can only concatenate list (not "tuple") to list

```

**Comment by McSwindler** [comment]
I was also seeing the same thing in my logs, found the cause was a rogue comma in the new generic parser.

Issued PR #57

**Comment by ljmerza** [comment]
fixed in v3.12.0
